tests/view/floors/test_floors.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+
+from http import HTTPStatus
+
+from tests.view.floors.mock_floors import FloorMock
+from timeless.restaurants.models import Floor
+from timeless.restaurants.floors.views import FloorListView
+
+""" Tests for floors.
+@todo #95:30min Continue implementation of 
```

Comment:
@paulodamaso This puzzle body is very small

Reply:
@ammaratef45 Sorry this was commited by mistake.

Reply:
@ammaratef45 Sorry, rough morning here